src/index.html

```diff
@@ -12,21 +12,34 @@ <h1>Leaderboard</h1>
     <div class="leaderboard">
         <div class="lb-header">
             <h2>Recent scores</h2>
-            <button type="button" class="refreshBtn">Refresh</button>
+            <div class="refreshBtn" title="Refresh"><i class="fas fa-sync-alt"></i></div>
         </div>
         <ul class="lb-list">
         </ul>
     </div>
     <div class="addscore">
+        <div class="addscore-container">
         <div class="as-header">
             <h2>Add your score</h2>
         </div>
         <form name="add-score" class="as-form">
-            <input name="name" type="text" placeholder="Your name" required />
-            <input name="score" type="number" placeholder="Your score" required />
-            <button type="submit">Submit</button>
+
+            <div class="form-group">
+                <span><i class="fas fa-user"></i></span>
+                <input class="form-field" name="name" type="text" placeholder="Your name" required />
+            </div>
+            <div class="form-group">
+                <span><i class="fas fa-poll"></i></span>
+                <input class="form-field" name="score" type="number" placeholder="Your score" required />
+            </div>
+            
+            <button type="submit" class="button-base ripple">Submit</button>
         </form>
     </div>
 </div>
+</div>
+
+<!-- Font Awesome 5 -->
```

Comment:
- [ ] OPTIONAL
There are comments found in your code on the repository. Kindly endeavor to remove **all** comments from your code before pushing it to the repository so as to keep your code well organized.